lib/hw_unique_key/Kconfig

```diff
@@ -4,6 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+config HW_UNIQUE_KEY_LOAD
+	bool "Load Hardware Unique key on boot"
+	depends on HAS_HW_NRF_ACL
+	depends on HAS_HW_NRF_CC310
+	depends on NRF_CC3XX_PLATFORM
+	select FPROTECT
```

Comment:
Why are we using `select` here? Could we instead update the definition of `FPROTECT` to have `default y if HW_UNIQUE_KEY_LOAD` and then have a `depends on FPROTECT` here instead?

Reply:
This gives a circular dependency. Left it as is for now.